data/roles/gecko_t_osx_1100_m1.yaml

```diff
@@ -30,7 +30,7 @@ packages_classes:
   - python3
   - python3_psutil
   - python3_zstandard
-  - rosetta_2
+#  - rosetta_2
```

Comment:
I think we still need to install rosetta on the m1's. (still x86 are: python installers (installs universal runtime), puppet-agent)

Reply:
This is true but forcing the install fails on non-arm hardware which is what the circleci vm instances run.  So we don't really have proper arch coverage on that platform for now.  I think it is acceptable to not include it here by the simple fact that installing puppet itself from the pkg requires rosetta2 to be installed.  So part of the bootstrapping process of an M1 is to install rosetta2 and then install puppet.

Reply:
:+1: good reasoning. we'll have installed it when bootstrapping, or have to reinstall it to apply puppet again if it gets removed